pkg/address/address.go

```diff
@@ -0,0 +1,22 @@
+package address
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+)
+
+// ConvertBech32Prefix convert bech32 address to specified prefix.
+func ConvertBech32Prefix(address, prefix string) (string, error) {
```

Comment:
Can you please add a test for this?

Reply:
Done

Reply:
By the way it is not 100% covered is it okay?